### ISSUE TITLE
Make screen vs. tmux configurable at runtime

### ISSUE THREAD

**Issue by sur5r** [DE-IBH/apt-dater#98]
Hi,

I was wondering wether you would be open to a patch that make the choice of screen vs. tmux configurable in apt-dater.xml instead of compile time.

I just wanted to ask beforehand just in case there's a reason not to do that.

Cheers,
sur5r

**Comment by liske** [comment]
Hi,

I will consider to merge such a patch. `tmux` should be default since this is IMHO the only way to support multi-user environments.

TIA,
Thomas

**Comment by sulicz** [comment]
+1

**Comment by sur5r** [comment]
As I'm currently not using apt-dater anymore I just wanted to note that I will not have the time to implement this in the foreseeable future. Just in case anyone else wants to give it a shot...

**Comment by henti** [comment]
Just to clarify, the ttmux (tmux and screen) is defined at compile time and cannot be changed once compiled, so binary packages supplied by distros enforce which muxer to use ?

**Comment by liske** [comment]
Yes it does and Debian uses tmux since it is the default and `screen` lacks features for multi-user setups. @henti
